st2actions/st2actions/runners/mistral/v2.py

```diff
@@ -245,6 +245,16 @@ def run(self, action_parameters):
         raise Exception('Failed to connect to mistral on %s. Make sure that mistral is running '
                         'and that the url is set correctly in the config.', self.url)
 
+    def cancel(self):
+        mistral_ctx = self.context.get('mistral', dict())
+
+        if not mistral_ctx.get('execution_id'):
+            raise Exception('Unable to cancel because mistral execution_id is missing.')
+
+        # There is no cancellation state in Mistral. Pause the workflow so
+        # actions that are still executing can gracefully reach completion.
+        self._client.executions.update(mistral_ctx.get('execution_id'), 'PAUSED')
```

Comment:
So if this is the case, I am not sure what the value of CANCEL is? It will still run to completion, right? In action chain, the ongoing execution is allowed to complete but the next tasks are not picked up.